tests/dort_test.py

```diff
@@ -0,0 +1,202 @@
+import copy
+import inspect
+import itertools
+import sys
+import unittest
+import warnings
+from typing import Optional
+
+import onnxruntime
+import torch
+import torch.nn as nn
+import torch.onnx
+from onnx.inliner import inline_local_functions
+
+
+def make_aot_ort(
+    dynamic: bool = False,
+    ort_optimization_level: Optional[str] = None,
+) -> tuple:
```

Comment:
```suggestion
) -> tuple:
```

Is the return type accurate?